mp/src/game/server/momentum/mom_chat_commands.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+struct command_alias_t
```

Comment:
This struct shouldn't be available to everyone that includes the file, keep it in the `.cpp`

Reply:
What if I use the struct as a parameter to RegisterCommand? They will need the header file either way so we could make use of it.

Reply:
Up to you how you want to implement this. If you end up exposing some function that uses the struct then feel free to keep it.